ccswbs/gus
# Add new top navigation to Content Hub pages

# Summary of changes
Add the new UG top navigation and associated web components to the Content Hub. Also remove the default submenu from pages and programs not assigned to an existing menu.

## Frontend
- Update `gatsby-ssr.js` with links to new web components and styles
- Update `headerMenu.js` to use new sub-navigation logic and structure
- Remove references to old `main` menu from `index.js`, `page.js` and `program-page.js`

### Incidental Changes
- Upgrade packages
- Remove jQuery (no longer needed)
- Increase timeout value for `gatsby-source-drupal` to prevent failed local builds
- Reduce WordPress content pulled in Dev environment to avoid timeout errors

## Backend
N/A

# Test Plan
- View https://chugtest.gatsbyjs.io/ and confirm the new Top Navigation appears and works as expected. No submenu should be on the page.
- Go to https://chugtest.gatsbyjs.io/studentexperience/ and confirm dropdown menu items work as expected. Also confirm the Modal Video works as expected.
- Go to https://chugtest.gatsbyjs.io/pathobiology/people/faculty/andrew-kropinski/ and confirm OVC's menu works as expected
- Go to https://chugtest.gatsbyjs.io/grce/ and confirm "GRCE" shows and works as this site's home link.
- Go to program https://chugtest.gatsbyjs.io/programs/bachelor-of-arts-general-fully-online/ and confirm no submenu appears
- Go to major page https://chugtest.gatsbyjs.io/programs/biomedical-engineering/ and confirm no submenu appears

# Known Issues
- Some units use the word "Home" for their top navigation link. This will need to be updated to a proper site name for better UX and information architecture.
- Sites with long menu titles look bad at the 1024px width breakpoint: 
![image](https://github.com/ccswbs/gus/assets/13698158/998cfd7a-f253-4a74-a178-0648eebb1a4a)
